extensions/ql-vscode/src/remote-queries/remote-query-info.ts

```diff
@@ -0,0 +1,15 @@
+
+// TODO This is a stub and will be filled implemented in later PRs.
+
+import { QueryStatus } from '../query-status';
+
+/**
+ * Information about a remote query.
+ */
+export interface RemoteQueryInfo {
```

Comment:
Appreciate you're following the same name as local queries, but I think a more correct name for this is something like `RemoteQueryHistoryItem`. It'd hold information needed for a remote query to be shown in the Query History. 

I'm just thinking that with the current state we'd have `RemoteQuery` and `RemoteQueryInfo` which is a bit confusing.

Reply:
Thanks for updating!